zapcli/zap_helper.py

```diff
@@ -394,5 +394,5 @@ def html_report(self, file_path):
     @staticmethod
     def _write_report(report, file_path):
         """Write report to the given file path."""
-        with open(file_path, 'w') as f:
+        with open(file_path, mode='wb') as f:
             f.write(report)
```

Comment:
I tested it locally with a Python 3.5 virtual environment. The build passes because they're only unit tests so they mock actually writing to a file. We could probably add some integration tests as well that might catch this sort of thing.